code/game/objects/items/ego_weapons/he.dm

```diff
@@ -167,7 +167,7 @@
 	return ..()
 
 /obj/item/ego_weapon/christmas
-	name = "christmas"
+	name = "merriment"		// I coded this ego, I will die on this hill. Christmas is a fucking dumb name.
```

Comment:
Infinite hatred and Christmas both fit the theme of the abnormality, it has nothing to do with joy, let alone merriment.